packages/admin-ui-docs/content/tooltip/overview.mdx

```diff
@@ -0,0 +1,74 @@
+import { Section } from '@site/src/components/Section'
+import { IntroSection } from '@site/src/components/IntroSection'
+
+<IntroSection>
+
+## What is this?
+
+Tooltips are a small informative pop-ups that appear when an item is being hovered, focused or touched. It implements [WAI-ARIA Tooltip Pattern](https://www.w3.org/TR/wai-aria-practices/#tooltip).
+
+</IntroSection>
+
+<Section>
+
+### Text
+
+```jsx live
+<Tooltip text="Tooltip text" />
+```
+
+</Section>
+
+<Section>
+
+### Target
+
+You can change the tooltip target through children.
```

Comment:
You will need to add the boolean prop `hasExplanation` to the <Section/> to let the text with the correct spacing between the title and the live preview code